src/Entity/User.php

```diff
@@ -50,6 +51,12 @@ class User implements UserInterface
      */
     private $homepage = "";
 
+
+    /**
+     * @ORM\Column(type="string", length=180, unique=true)
```

Comment:
Prie `form`os nurodai, kad laukas nėra privalomas, bet MySql lentelės pusėje jis vis dar (standartiškai) yra nurodytas, kaip privalomas.
https://github.com/aurelijusb/kickstart/blob/6dd2bb7240976b3b4726eb1809031d55fea11348/src/Form/RegistrationFormType.php#L38

Todėl neįvedus reik6mių registruojantis:
![image](https://user-images.githubusercontent.com/1453957/57979707-449f0c80-7a2a-11e9-8e6f-cea2b565b53c.png)

lūža duomenų bazės pusėje:
![image](https://user-images.githubusercontent.com/1453957/57979700-34872d00-7a2a-11e9-8b1a-8358e4ae4d3e.png)

Yra gera praktika, kad apsaugos būtų uždėtos formos pusėje – tada klaidos pranešimas būna labiau suprantamas ir naudotojas gali lengviau pasitaisyti pats (o ne skambinti į IT skyrių, kodėl jam išmetę klaidą su užrašu `500 Internal server error`)

Reply:
```suggestion
     * @ORM\Column(type="string", length=180)
```
Malonu, kad pagalvojai, jos `Linkedin` adresai turi būti unikalūs, bet tada reikėtų paruošti ir žmogiškesnius klaidos pranešimus:
![image](https://user-images.githubusercontent.com/1453957/57979746-ba0add00-7a2a-11e9-9841-ec2b10b112cd.png)
Nes `500 Internal server error` netechniniams žmonėms mažai sakys, kad jie padarė klaidą `Linkedin` adrese arba sistema neleidžia užsiregistruoti tam pačiam žmogui, bet turėti keli el. pašto adresus (pvz. nori testuoti su keliomis rolėmis)